schema.SQL

```diff
@@ -1,43 +1,47 @@
+-- psql command to run this file in terminal:
+-- \i /Users/noahbeito/Desktop/RFP2303/QuestionsAPI/schema.sql
+
 DROP TABLE IF EXISTS questions CASCADE;
 DROP TABLE IF EXISTS answers CASCADE;
 DROP TABLE IF EXISTS photos CASCADE;
 
-CREATE TABLE Questions (
+
+CREATE TABLE questions (
  id BIGSERIAL NOT NULL,
  product_id INTEGER NOT NULL,
  body TEXT NOT NULL,
- date_written TEXT NOT NULL,
+ date_written BIGINT NOT NULL,
  asker_name TEXT NOT NULL,
  asker_email TEXT,
  reported BOOLEAN,
  helpful INTEGER
 );
 
 
-ALTER TABLE Questions ADD CONSTRAINT Questions_pkey PRIMARY KEY (id);
+ALTER TABLE questions ADD CONSTRAINT questions_pkey PRIMARY KEY (id);
```

Comment:
I don't think you need to use Alter Table syntax for the primary keys. This might be useful for foreign keys, but I am pretty sure the imported data recognizes the first id as the primary key. At least, I defined my primary key in my schema creation (as opposed to adding an alter table statement) and didn't have any problem with it importing my data.